src/index.ts

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+import formats from './formats';
+import states from './generated/states';
+import parse from './parse/parse';
+
+export default {
```

Comment:
@lpatiny 
This is a breaking change (and I think a wrong one).
Now you have to do `import mrz from 'mrz'; mrz.parse()` instead of `import { parse } from 'mrz'`

Reply:
isn't it the same thing?

Reply:
Thank you, you explained it in the other review